src/pudl/transform/ferc714.py

```diff
@@ -421,7 +430,8 @@ def demand_hourly_pa(tfr_dfs):
 
     # Clean UTC offset codes
     df["utc_offset_code"] = df["utc_offset_code"].str.strip().str.upper()
-    df["utc_offset_code"] = df.pipe(_standardize_offset_codes, OFFSET_CODE_FIXES)
+    df["utc_offset_code"] = df.pipe(
+        _standardize_offset_codes, OFFSET_CODE_FIXES)
```

Comment:
Here's another line that appears to have been auto formatted to 79 instead of 88 characters.